src/core/bitop.d

```diff
@@ -72,20 +72,17 @@ unittest
  */
 int bsf(size_t v) pure;
 
+version(CoreDdoc)
 /// ditto
+int bsf(ulong v) pure;
+
+version(D_LP64) { } else
```

Comment:
`static if (size_t.sizeof != ulong.sizeof)` seems more appropriate to me, since the point is, literally, to detect when the above `int bsf(size_t v)` is not equivalent to `int bsf(ulong v)`.

Also, `static if` is used for this purpose everywhere else in this file; this is a gratuitous inconsistency.